retire-js/RetireJsVulns.json

```diff
@@ -559,6 +559,7 @@
       "jquery-ui",
       "jquery.ui"
     ],
+    "npmname": "jquery-ui",
```

Comment:
Do we know what this `npmname` property does? I can't imagine it's terribly relevant to us, but still curious.